app/views/testing_grounds/import.html.haml

```diff
@@ -11,10 +14,12 @@
 = form_for(@import, url: import_testing_grounds_path, method: :post) do |form|
   .form-group
     = form.label :topology_id
+    = tooltip_tag('amazin')
```

Comment:
I'm sure it is. :D 

Is it possible to remove the tooltips for which there's no help text? Same goes for `topologies/_form.html.haml`.